docs/en/advanced_guides/conventions.md

```diff
@@ -2,6 +2,10 @@
 
 Please check the following conventions if you would like to modify MMDetection as your own project.
 
+## Image pipeline input shape
+
+The image pipeline input shape is `(weight, height)`, and the shape of the pipeline output is `(height, weight)` in OpenMMLab 2.0.
```

Comment:
Should be more specific and give several examples.
In model, the shape is also always (height, width)